EPolicija/EPolicija.csproj

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why create new projects, this complicates compiling, and needlessly creates multiple files

Reply:
I'll wrap projects